cluster/definition.go

```diff
@@ -38,6 +38,13 @@ func WithVersion(version string) func(*Definition) {
 	}
 }
 
+// WithDKGAlgorithm returns an option to set a non-default DKG algorithm in a new definition.
+func WithDKGAlgorithm(algorithm string) func(*Definition) {
```

Comment:
why is this needed? we only support one dkg algo atm

Reply:
In createdkg.go it populates opts.. which we need to extend WithVersion.

Reply:
but why do we need to populate the dkg algorithm?

Reply:
I think this is to respect the given configuration. I believe the implementation assumes different algorithms passed from the config. If you think this is odd, then we can have a ticket to remove this field from the config entirely.